core/test/integration/model/model_posts_spec.js

```diff
@@ -457,20 +474,21 @@ describe('Post Model', function () {
                 }).catch(done);
             });
 
-            it('can change markdown to number', function (done) {
-                var postId = testUtils.DataGenerator.Content.posts[0].id;
-
-                PostModel.findOne({id: postId}).then(function (results) {
-                    should.exist(results);
-                    var post = results.toJSON();
-                    post.title.should.not.equal('123');
-                    return PostModel.edit({markdown: 123}, _.extend({}, context, {id: postId}));
-                }).then(function (edited) {
-                    should.exist(edited);
-                    edited.attributes.markdown.should.equal('123');
-                    done();
-                }).catch(done);
-            });
+            // TODO: is a similar test needed for mobiledoc?
```

Comment:
@kirrg001 can this and the other commented test below be removed? The `mobiledoc` field can't be a plain number so I don't think these tests are relevant any more